bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -9,6 +9,9 @@ const presenter = {
   subscribePoll: true,
   subscribeAnswers: true,
 
+  // presentation
+  removePresentation: true,
+  sharePresentation: true,
```

Comment:
And the name is share because thats the event name we publish to redis. `share_presentation`